client/go/internal/cli/cmd/visit.go

```diff
@@ -0,0 +1,339 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// vespa visit command
+// Author: arnej
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/vespa-engine/vespa/client/go/internal/util"
+	"github.com/vespa-engine/vespa/client/go/internal/vespa"
+)
+
+type visitArgs struct {
+	contentCluster string
+	fieldSet       string
+	selection      string
+	makeFeed       bool
+	jsonLines      bool
+	pretty         bool
+	quietMode      bool
+	chunkCount     int
```

Comment:
In https://github.com/vespa-engine/sample-apps/pull/1127 is was a little surprise this was not called wantedDocumentCound as in the documentation - confusing

Reply:
but chunkCount is a better name, yes :-)